CONTRIBUTING.md

```diff
@@ -32,6 +32,74 @@ Here are a few guidelines to follow when submitting a pull request:
 
 Have questions or need help with setup? Open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
 
+### Confirming your environment is prepared
+
+#### Confirm Ruby
+
+To serve the site locally, we will need Ruby. Confirm Ruby is installed:
+
+```sh
+gem -v
+```
+
+If you do not see a version number, go to the [RubyGems download page](https://rubygems.org/pages/download) to install Ruby.
+
+#### Confirm Ruby Bundler
+
+```sh
+bundle -v
+```
+
+If you do not see a version number, and you have confirmed Ruby:
+
+```sh
+gem install bundler
+```
+
+#### Confirm NPM
+
+To install the required modules, we will need Node Package Manager (NPM). Confirm NPM is available:
+
+```sh
+npm -v
+```
+
+If you do not see a version number, go to the [Node.js download page](https://nodejs.org/en/download/) to install Node.js.
```

Comment:
Interesting, I believe we have hit a knowledge gap for me with regard to Node. It sounds like there are three possibilities:

1. Something I did not know existed `nvm` - and don't know how to write to that without a reference to what it is and how it works with Terminal/Shell.
2. Terminal/Shell + `sudo` to make global.
3. Terminal/Shell *without* `sudo`: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md

Sound about right to you?